vortex-btrblocks/src/float/stats.rs

```diff
@@ -175,6 +201,11 @@ where
     let value_count = value_count
         .try_into()
         .vortex_expect("null_count must fit in u32");
+    let distinct_values_count = if count_distinct_values {
+        distinct_values.len().try_into().vortex_unwrap()
+    } else {
+        u32::MAX
+    };
```

Comment:
i thought there was a reason i did this in the inner loop but now i can't remember why, so maybe it's unnecessary now

Reply:
I couldn’t figure out why this had to be recomputed on each loop